tools/run_python_system_tests.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import pip
+import time
+import json
+import zipfile
+import argparse
+import tempfile
+import urllib.request
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-d', '--driver', default='niswitch', type=str)
```

Comment:
You should make this required and not have a default.

Reply:
You should also add some high-level documentation to argparse about what this tool does, so the information is printed when invoked with --help, which is the expectation of any command line tool user.

Reply:
Fixed